### PR TITLE
[wasm][debugger] Initial support for proxying flat sessions

### PR DESCRIPTION
**Pull request by lewing**
Initial support means handling the browser endpoint and proxying any sessionId values, along with making sure we load pdbs as things start initializing

### CODE REVIEW

File: sdks/wasm/Mono.WebAssembly.DebuggerProxy/DebugStore.cs [hunk 1]
```diff
@@ -298,34 +292,38 @@ internal class AssemblyInfo {
...
-				}
 
 				rp.ReadingMode = ReadingMode.Immediate;
 				rp.InMemory = true;
 
 				this.image = ModuleDefinition.ReadModule (new MemoryStream (assembly), rp);
 			} catch (BadImageFormatException ex) {
 				Console.WriteLine ($"Failed to read assembly as portable PDB: {ex.Message}");
+			} catch (ArgumentNullException) {
+				if (pdb != null)
```
Comment (kg) [inline sdks/wasm/Mono.WebAssembly.DebuggerProxy/DebugStore.cs]: Should we print a message if pdb is null here?

Comment (steveisok) [reply sdks/wasm/Mono.WebAssembly.DebuggerProxy/DebugStore.cs]: From what I see, I think so.

Comment (lewing) [reply sdks/wasm/Mono.WebAssembly.DebuggerProxy/DebugStore.cs]: I was avoiding that for the embedded pdb case

**Comment by lewing** [comment]
@monojenkins build Linux ARMv7

**Comment by lewing** [comment]
@monojenkins build Linux ARMv7

**Pull request merged**